env/env.go

```diff
@@ -10,8 +10,14 @@ func IsAppEngine() bool {
 	return set
 }
 
+// Extract google cloud project from environment
```

Comment:
Go comments must start with the func name.

i.e, // GoogleCloudProject extracts ths ....

Reply:
Already fixed in the recent rebase 👍